locations/spiders/amigo_us.py

```diff
@@ -0,0 +1,12 @@
+from locations.storefinders.agile_store_locator import AgileStoreLocatorSpider
+
+
+class AmigoUSSpider(AgileStoreLocatorSpider):
```

Comment:
Just need to change `US` -> `PR` (Puerto Rico is a different ISO 3166 country code)

Reply:
Ah, wasn't too sure given it's a US territory.